Tasks/ServiceFabricPowerShellV1/ServiceFabricPowerShell.ps1

```diff
@@ -29,10 +29,10 @@ try
 {
     # Initialize Service Fabric.
     Import-Module $PSScriptRoot\ps_modules\ServiceFabricHelpers
-    $connectedServiceEndpoint = Get-VstsEndpoint -Name $serviceConnectionName -Require
 
+    $global:operationId = $SF_Operations.Undefined
```

Comment:
why undefined?

Reply:
Shouldn't it be connect?

Reply:
it is set to connect in Connect helper Connect-ServiceFabricClusterFromServiceEndpoint.ps1